src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    public static final String URL = "YOUR DATABASE URL";
-    public static final String USERNAME = "YOUR USERNAME";
-    public static final String PASSWORD = "YOUR PASSWORD";
-    public static final String JDBC_DRIVER = "YOUR DRIVER";
+    public static final String DRIVERS_URL = "jdbc:mysql://127.0.0.1:3306/driver_db";
```

Comment:
```suggestion
    public static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/driver_db";
```
it's not only about drivers